frontend/public/static/locale/fr/translate.ftl

```diff
@@ -35,6 +35,15 @@ machinery-translation-number-occurrences =
     .title = Nombre d'occurrences de la traduction
 
 
+## Entity Navigation
+## Shows next/previous buttons.
+
+entitynavigation-next = <glyph></glyph>Suivant
+    .title = Go To Next String (Alt + Down)
+entitynavigation-previous = <glyph></glyph>Précédent
+    .title = Go To Previous String (Alt + Up)
```

Comment:
Hi @adngdb, Any chance you could translate the `.title` attributes to French? :)

Reply:
_Aller à la chaîne précédente (Alt + Haut)_